configuration/config2/CfgStringLimits.cpp

```diff
@@ -20,28 +20,7 @@
 
 std::string CfgStringLimits::getString() const
 {
-
-    std::string limitStr = "";
-
-    if (m_length != 0)
-    {
-        //limitStr += "string " + m_length + " charcters in length.";
-    }
-    else 
-    {
-        //limitStr += "string between " + getMin() + " and ";
-        if (getMax() == INT_MAX)
-        {
-            limitStr += " unlimited ";
-        }
-        else
-        {
-            limitStr += getMax();
-        }
-        limitStr += "characters in length.";
-
-    }
-
+    std::string limitStr = "";  // this will turn into an error string describing why a value is invalid
```

Comment:
I'm having a hard time understanding the usefulness of this function, is it meant to be overwritten?
also, if the logic is complete, could we just return ""; ?